README.md

```diff
@@ -46,6 +46,10 @@ Create a personal access token and provide to the VSTS CLI via the login command
 vsts login --instance https://MYACCOUNT.visualstudio.com --token MYTOKEN
 ```
 
+### Using VSTS CLI Behind a Proxy
+
+The vsts cli picks the network proxy configuration from the environment variables `HTTP_PROXY`, and `HTTPS_PROXY`. 
```

Comment:
Very minor but the vsts-cli is always referred to as `VSTS CLI` in the [README.md](https://github.com/Microsoft/vsts-cli/blob/dev/README.md)